java/java.freeform/src/org/netbeans/modules/java/freeform/jdkselection/jdk.xml

```diff
@@ -159,7 +159,10 @@
         <!-- Define fallback values of some things. -->
         <property name="java.home.parent" location="${java.home}/.."/>
         <condition property="nbjdk.home" value="${java.home.parent}">
-            <available file="${java.home.parent}/lib/tools.jar" type="file"/>
+            <available file="${java.home.parent}/jmods/java.base.jmod" type="file"/>
```

Comment:
My though was this (this is the original line 162) in combination with the addtion lines 163-165 you added:

```suggestion
            <available file="${java.home.parent}/lib/tools.jar" type="file"/>
```